projects/core/src/i18n/i18next/i18next-init.ts

```diff
@@ -76,3 +84,34 @@ export function i18nextGetHttpClient(
       );
   };
 }
+
+/**
+ * Resolves the relative path to the absolute one in SSR, using the server request's origin.
+ * It's needed, because Angular Universal doesn't support relative URLs in HttpClient. See Angular issues:
+ * - https://github.com/angular/angular/issues/19224
+ * - https://github.com/angular/universal/issues/858
+ */
+export function getLoadPath(
+  path: string = '',
+  platform: string,
+  serverRequestOrigin: string = ''
+): string {
+  if (!path) {
+    return undefined;
+  }
+  if (
+    isPlatformServer(platform) &&
```

Comment:
do we necessarily need platform here? as we don't get serverRequestOrigin in a browser by design.